src/tests/ftest/unittest/Unittest.py

```diff
@@ -45,9 +45,10 @@ def unittest_runner(self, unit_testname):
         """
         name = self.params.get("testname", '/run/UnitTest/{0}/'
                                .format(unit_testname))
+        server = self.params.get("test_machines", "/run/hosts/*")
         bin_path = get_file_path(name, "install/bin")
 
-        cmd = ("{0}".format(bin_path[0]))
+        cmd = ("ssh {} {}".format(server[0], bin_path[0]))
```

Comment:
Why not use the new ClusterShell support being added?

Reply:
yes was thinking of doing it but want to see if test-tag is picking and it's working in Ci..

Reply:
Seems it is:
```
01:24:30.725  Detected tests:  
01:24:30.725  ./daos_test/daos_core_test.py  
01:24:30.725  ./unittest/Unittest.py
```

and https://build.hpdd.intel.com/job/daos-stack/job/daos//view/change-requests/job/PR-641/2/testReport/(root)/UnitTest/